model/database/registry.yaml

```diff
@@ -543,7 +543,11 @@ groups:
         stability: experimental
         brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
         examples: ["Eventual", "ConsistentPrefix", "BoundedStaleness", "Strong", "Session"]
-
+      - id: db.cosmosdb.regions_contacted
+        type: string
```

Comment:
this feels like it should be a `string[]`

I'm not sure if we've put any `string[]` attributes on metrics yet though...

I've added this to Monday's semconv SIG agenda to get more feedback in case we don't get more feedback here (and I'm off the rest of this week)

Reply:
how does it matter? if its string or string[]

Reply:
since the underlying data is a list, it should ideally be modeled that way (instead of encoded into a comma-separated string)

e.g. see `http.request.header.<key>`

Reply:
Unresolving this, as discussed it should be an array.